src/app/lib/helpers.test.ts

```diff
@@ -0,0 +1,9 @@
+import { parseNumberToBigInt } from './helpers'
+
+describe('parseNumberToBigInt', () => {
+  it('should return a value of type bigint', () => {
+    expect(parseNumberToBigInt(33333.3)).toEqual(33333300000000n)
+    expect(parseNumberToBigInt(2.0121)).toEqual(2012100000n)
+    expect(parseNumberToBigInt(5)).toEqual(5000000000n)
```

Comment:
```suggestion
    expect(parseNumberToBigInt(33333.3)).toEqual(33333_300_000_000n)
    expect(parseNumberToBigInt(2.0121)).toEqual(2_012_100_000n)
    expect(parseNumberToBigInt(5)).toEqual(5_000_000_000n)
```

something like this, so it's easier to eyeball where 10^9 is.